### DISCUSSION TITLE
Github Action not building and no logs are visible

### DISCUSSION THREAD

**Discussion by Skarlso** [community/community#26131] [Actions]
<p>Hi everyone.</p>
<p>I’m trying to use Github Actions, but I’m getting a failing build without much output.</p>
<p>This is all the output:</p>
<pre><code>### STARTED build_it 05:42:03Z

Pulling image: gcr.io/github-actions-images/action-runner:latest
latest: Pulling from github-actions-images/action-runner
169185f82c45: Pulling fs layer
0ccde4b6b241: Pulling fs layer
d0372f57daa2: Pulling fs layer
165911d108d6: Pulling fs layer
54996bce1de5: Pulling fs layer
165911d108d6: Waiting
54996bce1de5: Waiting
d0372f57daa2: Verifying Checksum
d0372f57daa2: Download complete
169185f82c45: Verifying Checksum
169185f82c45: Download complete
0ccde4b6b241: Verifying Checksum
0ccde4b6b241: Download complete
165911d108d6: Verifying Checksum
165911d108d6: Download complete
54996bce1de5: Verifying Checksum
54996bce1de5: Download complete
169185f82c45: Pull complete
0ccde4b6b241: Pull complete
d0372f57daa2: Pull complete
165911d108d6: Pull complete
54996bce1de5: Pull complete
Digest: sha256:c9bb432ec5ec08ee08b040a9fccacebbbf8a91444dac4721600cf5dca9dae57e
Status: Downloaded newer image for gcr.io/github-actions-images/action-runner:latest

### FAILED build_it 05:42:03Z (530ms)
</code></pre>
<p>Here is the repository: <a href="https://github.com/Skarlso/blogsource" rel="nofollow noopener">https://github.com/Skarlso/blogsource</a></p>
<p>And here is the workflow file:</p>
<pre><code>workflow "Publish Blog" {
  on = "push"
  resolves = ["publish_it"]
}

action "build_it" {
  uses = "./.github/actions/build_it@master"
  secrets = ["GITHUB_TOKEN"]
}

action "publish_it" {
  uses = "./.github/actions/publish_it@master"
  needs = ["build_it"]
  secrets = ["GITHUB_TOKEN"]
}
</code></pre>
<p>And here are the actions: <a href="https://github.com/Skarlso/blogsource/tree/master/.github" rel="nofollow noopener">https://github.com/Skarlso/blogsource/tree/master/.github</a></p>
<p>At this point I have no idea what the problem might be. :confused:</p>
<p>Any help would be greatly appreciated!</p>
<p>Cheers,</p>
<p>Gergely.</p>

**Comment by Skarlso** [comment, marked as answer]
<p>The problem was that it didn’t actually find the action to execute. I had to extract the actions into their respective repositories. Now it works.</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hey <a class="mention" href="/skarlso">@skarlso</a> thanks a lot for coming back to share what worked for you! In the meantime, we’re looking forward to seeing you around!</p>
